Content.Server/GameObjects/Components/Mobs/State/DeadMobState.cs

```diff
@@ -33,7 +34,10 @@ public override void EnterState(IEntity entity)
 
             if (entity.TryGetComponent(out IPhysBody? physics))
             {
-                physics.CanCollide = false;
+                // remove mob from layers where they can be hit by ranged weapon
+                // other layers will stay unchanged
+                foreach (var fixture in physics.Fixtures)
+                    fixture.CollisionLayer &= (int) ~CollisionGroup.WeaponMask;
```

Comment:
Should get a specific fixture IMO.

Reply:
By fixture name?

Reply:
Yeah probably, just add a name to mobs for their default circle fixture and refer to it by that name.